platform/rct/hal/usb_stm32/USB-CDC_device_stm32.c

```diff
@@ -1,52 +1,250 @@
-#include "USB-CDC_device.h"
```

Comment:
If you are using emacs (which I think you are) highlight the includes and issue `sort-lines`